components/constants.js

```diff
@@ -0,0 +1,5 @@
+const CONSTANTS = {
+    site_title: 'Awe & Reverence',
```

Comment:
do this:

```

const SITE_TITLE = 'Awe & Reverence';

const PAGE_TITLES = {
    '/': 'Home',
    ...
};

const CONSTANTS = {
    SITE_TITLE,
    PAGE_TITLES,
};

export default CONSTANTS;

Reply:
Alternatively, a more succinct solution:

```
export const SITE_TITLE = 'Awe & Reverence';

export const PAGE_TITLES = {
    '/': 'Home',
    ...
};
```

And at the import site, just import what you need.

```
import { PAGE_TITLES, SITE_TITLE } from 'path/to/constants';
```